### ISSUE TITLE
Strange results

### ISSUE THREAD

**Issue by bristoljon** [potatopotaro/faunadb-geo#2]
Hello again,

Now this may well be something my side but I'm getting some weird results when using the Geosearch query builder.

In short, if I use a large radius of 10km (10000m) I see all the matches I'd expect to see but if I reduce the radius to a point still greater than the distance of some of my results, they all disappear and I get no matches.

I know this is going to be tricky to debug but I can't see any issue with the numbers I'm passing in and so raising this in case you can point me in the best direction and places to step through in your code.

It's really odd actually, almost random. As I reduce the radius by 1km it's kind of 50:50 if I get results or not.

<img width="560" alt="Screenshot 2020-07-08 at 20 54 48" src="https://user-images.githubusercontent.com/13088751/86964994-cbe39e00-c15e-11ea-9d1f-7bb237e33716.png">
<img width="580" alt="Screenshot 2020-07-08 at 20 54 36" src="https://user-images.githubusercontent.com/13088751/86964995-cbe39e00-c15e-11ea-8f94-c9e48d96b8dc.png">
<img width="571" alt="Screenshot 2020-07-08 at 20 54 22" src="https://user-images.githubusercontent.com/13088751/86964997-cbe39e00-c15e-11ea-83ed-ea16eef1a3bd.png">

**Comment by potatopotaro** [comment]
Hey @bristoljon, how many cells are you using? Sometimes, if the number of cells is low, the differences in queries can be very sharp. You can probably verify if this is the case by comparing the results from your original 10km GeoSearch to the reduced radius GeoSearch that's acting up, **but**, dramatically increase the number of cells used for the latter GeoSearch.

Additionally, could you copy-paste the input and output logs for the above screenshots, please? Thanks again for using this package, I'll make sure to get you successfully settled in with it. Worst case scenario we can hop on a quick call to diagnose it. Cheers.

**Comment by bristoljon** [comment]
Hey, sorry for the delay responding. I did quickly try changing the maxReadOps value to no effect however I think I've figured it out now.

I just noticed in your readme you provide a `CreateGeoSpatialIndex` util. I had already created my own following your awesome list article and it was only returning a ref, not the geohash in values.

Using an index created by your helper seems to have fixed the issue. It breaks the intersection I'm using to filter by other fields but that should be easily fixed, I guess, by returning the same values in all the indexes used.

Anyway think can close this one now. Thanks again!
